src/coreclr/nativeaot/Runtime/i386/GcProbe.asm

```diff
@@ -261,16 +261,16 @@ WaitForGC:
         or          ecx, DEFAULT_PROBE_SAVE_FLAGS + PTFF_SAVE_RAX
         jmp         RhpWaitForGC
 
-FASTCALL_ENDFUNC
+_RhpGcProbeHijack@0  endp
 
 ifdef FEATURE_GC_STRESS
-FASTCALL_FUNC RhpGcStressHijack, 0
+_RhpGcStressHijack@0  proc public
 
         HijackFixupProlog
         or          ecx, DEFAULT_PROBE_SAVE_FLAGS + PTFF_SAVE_RAX
         jmp         RhpGcStressProbe
 
-FASTCALL_ENDFUNC
+_RhpGcStressHijack@0  endp
 
 FASTCALL_FUNC RhpHijackForGcStress, 0
```

Comment:
```suggestion
_RhpHijackForGcStress@0  proc public
```
This is currently unused, but I think it should match

Reply:
I was not sure if this would be called from C or asm. 

I hope to enable GC stress for NativeAOT in some form one day. 
Perhaps not in the form it was, as a lot of things changed with switch to RyuJIT and CoreCLR-like suspension plan. However, some of this code may still be useful at some point, so I try to keep it around, even if unused.